UnleashClient/__init__.py

```diff
@@ -185,7 +184,32 @@ def __init__(
 
         self.metrics_headers: dict = {}
 
-        # Scheduler bootstrapping
+        self._init_scheduler(scheduler, scheduler_executor)
+
+        if custom_strategies:
+            self.engine.register_custom_strategies(custom_strategies)
+
+        self.strategy_mapping = {**custom_strategies}
+
+        # Client status
+        self.is_initialized = False
+
+        # Bootstrapping
+        if self.unleash_bootstrapped:
+            OfflineConnector(
+                engine=self.engine,
+                cache=self.cache,
+                scheduler=None,
+            ).load_features()
+
+        self.connector: Union[OfflineConnector, PollingConnector] = None
```

Comment:
We went through all that work of making the connector types swappable. Let's indicate that with the type hinting (you'll need to import this if you take the suggestion)

```suggestion
        self.connector: Union[BaseConnector] = None
```